<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>963</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,100,0">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0,0,0,0">
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Device</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="combo_device">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="keyboard_mode">
          <property name="text">
           <string>KeyBoard Input Mode</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="flip">
          <property name="text">
           <string>Flip</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Loading&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11"/>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_13">
                <item>
                 <widget class="QLabel" name="label_info2">
                  <property name="text">
                   <string>地图/功能测试</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="radioButton">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>目前仅支持山脊</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_get_map">
                  <property name="text">
                   <string>get map</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_reset_map">
                  <property name="text">
                   <string>reset map</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_save_map">
                  <property name="text">
                   <string>save map</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_move_up">
                  <property name="text">
                   <string>move up</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_move_down">
                  <property name="text">
                   <string>move down</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_move_left">
                  <property name="text">
                   <string>move left</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_move_right">
                  <property name="text">
                   <string>move right</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_move_stop">
                  <property name="text">
                   <string>move stop</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_attack">
                  <property name="text">
                   <string>test_attack</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_test_skill">
                  <property name="text">
                   <string>test skill</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_cursor">
                  <property name="text">
                   <string>show_cursor</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_15">
                <item>
                 <widget class="QLabel" name="label_info1">
                  <property name="text">
                   <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;使用条款：&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;本程序仅供测试学习使用，不针对任何程序和主体。&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;禁止任何形式的非法用途和盈利活动，&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;任何未经允许的传播和盈利与作者本人无关。&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;开始使用下方的任一功能，即视为认同上述条款！&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_reload">
                  <property name="text">
                   <string>RELOAD CONFIG</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_back">
                  <property name="text">
                   <string>RESET GAME</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_start">
                  <property name="text">
                   <string>START DETECTION</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="button_stop">
                  <property name="text">
                   <string>STOP DETECTION</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
        <property name="spacing">
         <number>3</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QTextEdit" name="textEdit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="autoFillBackground">
           <bool>false</bool>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_fps">
              <property name="text">
               <string>frame rate</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <widget class="QLabel" name="label_info_freq">
                  <property name="text">
                   <string>检测频率（越小越好，但吃性能）</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_6">
                  <item>
                   <widget class="QSlider" name="slider_frame_req">
                    <property name="minimum">
                     <number>1</number>
                    </property>
                    <property name="maximum">
                     <number>30</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_info_frame_freq">
                    <property name="text">
                     <string>检测频率</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_8">
                <item>
                 <widget class="QLabel" name="label_info_enemy_x">
                  <property name="text">
                   <string>攻击x方向阈值</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_13">
                  <item>
                   <widget class="QSlider" name="slider_attx">
                    <property name="minimum">
                     <number>100</number>
                    </property>
                    <property name="maximum">
                     <number>800</number>
                    </property>
                    <property name="singleStep">
                     <number>40</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_attx">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_7">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>攻击y方向阈值</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_12">
                  <item>
                   <widget class="QSlider" name="slider_atty">
                    <property name="minimum">
                     <number>150</number>
                    </property>
                    <property name="maximum">
                     <number>800</number>
                    </property>
                    <property name="singleStep">
                     <number>40</number>
                    </property>
                    <property name="orientation">
                     <enum>Qt::Horizontal</enum>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_atty">
                    <property name="text">
                     <string>TextLabel</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
